server/datasets/ingest.py

```diff
@@ -95,16 +99,19 @@ def _log(*args):
             msg = str(args[0])
         girder.logger.info(msg)
```

Comment:
Do you think it would be good to log this to the job as well?  By removing the `@staticmethod` and adding 
```
        if self.job:
            ModelImporter.model('job', 'jobs').updateJob(
                job=self.job,
                log=msg + '\n'
            )
```